modules/library/main/src/main/java/org/geotools/filter/function/FilterFunction_strURLEncode.java

```diff
@@ -0,0 +1,62 @@
+package org.geotools.filter.function;
```

Comment:
Added header.

Renamed class, and moved to org.geotools.filter.function.string package from org.geotools.filter.function